### ISSUE TITLE
Issue with js plugins

### ISSUE THREAD

**Issue by EtienneBourgeois56** [emlyon/Designing-With-Web#21]
I try to include a plugin from Jquery, following the indication on the page but i'm unable to suceed... the name of the plugin is pagepiling and there is my HTML code (the js file is after!):

<html lang="en">
   <div class="row">
  <head>
    <title>Djadja Website
    </title>
    
    <link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Dosis" rel="stylesheet">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css">
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" type="text/css" href="jquery.pagepiling.css" />

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    
<script type="text/javascript" src="jquery.pagepiling.js"></script>
    
    <style>
      body {
        background: black;
        height: 100vh; /
        display: grid; /
      }
      
      h1 {
        color: white;
        margin: auto;
        font-family: 'Oswald', cursive;
        font-size: 40px;
        text-align: center;
      }
      h3 { color: white;}
      
      body2 {
        font-family: "Dosis" , cursive;}
      
    </style>
  </head> 
  
     <div id="pagepiling">
  <body>
    <div class="container">
    <div class="row">
      <div class="ten columns">
        <div class="section">
    <h1>THE WOLF AND THE KID</h1>
    <h3> 
      There was once a little Kid whose growing horns made him think he was a grown-up Billy Goat and able to take care of himself. So one evening when the flock started home from the pasture and his mother called, the Kid paid no heed and kept right on nibbling the tender grass. A little later when he lifted his head, the flock was gone.
<p>
      He was all alone. The sun was sinking. Long shadows came creeping over the ground. A chilly little wind came creeping with them making scary noises in the grass. The Kid shivered as he thought of the terrible Wolf. Then he started wildly over the field, bleating for his mother. But not half-way, near a clump of trees, there was the Wolf! 
      </p>
<p>  The Kid knew there was little hope for him 
      "Please, Mr. Wolf," he said trembling, "I know you are going to eat me. But first please pipe me a tune, for I want to dance and be merry as long as I can."
</p>
  <p> The Wolf liked the idea of a little music before eating, so he struck up a merry tune and the Kid leaped and frisked gaily.
Meanwhile, the flock was moving slowly homeward. In the still evening air the Wolf's piping carried far. The Shepherd Dogs pricked up their ears. They recognized the song the Wolf sings before a feast, and in a moment they were racing back to the pasture. The Wolf's song ended suddenly, and as he ran, with the Dogs at his heels, he called himself a fool for turning piper to please a Kid, when he should have stuck to his butcher's trade.
  </p>
      </h3>
        
     <div style="text-align: center">
    <img src="https://media.giphy.com/media/yEStAKFg8EA3S/giphy.gif">
    </div>
        </div>
      </div>
    </div>
      </div>
    <div class="section">
  <h1>THE TORTOISE AND THE DUCKS
     
    </h1>
<h3>
  <div id="col">
    <div style="text-align: center">The Tortoise, you know, carries his house on his back. No matter how hard he tries, he cannot leave home. They say that Jupiter punished him so, because he was such a lazy stay-at-home that he would not go to Jupiter's wedding, even when especially invited.

After many years, Tortoise began to wish he had gone to that wedding. When he saw how gaily the birds flew about and how the Hare and the Chipmunk and all the other animals ran nimbly by, always eager to see everything there was to be seen, the Tortoise felt very sad and discontented. He wanted to see the world too, and there he was with a house on his back and little short legs that could hardly drag him along.

One day he met a pair of Ducks and told them all his trouble.

"We can help you to see the world," said the Ducks. "Take hold of this stick with your teeth and we will carry you far up in the air where you can see the whole countryside. But keep quiet or you will be sorry."

The Tortoise was very glad indeed. He seized the stick firmly with his teeth, the two Ducks took hold of it one at each end, and away they sailed up toward the clouds.

Just then a Crow flew by. He was very much astonished at the strange sight and cried:

"This must surely be the King of Tortoises!"

"Why certainly——" began the Tortoise.

But as he opened his mouth to say these foolish words he lost his hold on the stick, and down he fell to the ground, where he was dashed to pieces on a rock.
  </div>
        
  </div>
    </h3>
    </div>

    <h1>THE YOUNG CRAB AND HIS MOTHER
    
    </h1>
<h3>"Why in the world do you walk sideways like that?" said a Mother Crab to her son. "You should always walk straight forward with your toes turned out."
"Show me how to walk, mother dear," answered the little Crab obediently, "I want to learn."
So the old Crab tried and tried to walk straight forward. But she could walk sideways only, like her son. And when she wanted to turn her toes out she tripped and fell on her nose.
    </h3>
    <script src="cool-file.js"></script>
 </body>
       </div>
  </div>
  <clicbutton>
<div style="text-align: center">
<script src="cool-file.js"> 
</script>
<button onclick="showresult()">
  <a href="https://fr.wikipedia.org/wiki/%C3%89sope">Clic to know more about Aesop</a>
</button>
                            </div>
  </clicbutton>
</html>

There is the js file:
$(document).ready(function() {
	$('#pagepiling').pagepiling();
});

**Comment by MAKIO135** [comment]
Je crois qu'on s'est vu. Peu-tu fermer l'issue si c'est le cas.
